### PR TITLE
Update Alpine Linux Synapse Package Maintainer within installation.md

### PR DESCRIPTION
**Pull request by jahway603**
Update Alpine Linux Synapse Package Maintainer within installation.md as it is outdated.

### Pull Request Checklist

* [x] Pull request is based on the develop branch
* [N/A] Pull request includes a [changelog file](https://element-hq.github.io/synapse/latest/development/contributing_guide.html#changelog). The entry should:
  - Be a short description of your change which makes sense to users. "Fixed a bug that prevented receiving messages from other servers." instead of "Moved X method from `EventStore` to `EventWorkerStore`.".
  - Use markdown where necessary, mostly for `code blocks`.
  - End with either a period (.) or an exclamation mark (!).
  - Start with a capital letter.
  - Feel free to credit yourself, by adding a sentence "Contributed by @github_username." or "Contributed by [Your Name]." to the end of the entry.
* [x] [Code style](https://element-hq.github.io/synapse/latest/code_style.html) is correct
  (run the [linters](https://element-hq.github.io/synapse/latest/development/contributing_guide.html#run-the-linters))

### CODE REVIEW

**Review by MadLittleMods — COMMENTED** [review]
@jahway603 This is just waiting on the CLA being signed

File: docs/setup/installation.md [hunk 1]
```diff
@@ -157,7 +157,7 @@ sudo pip install py-bcrypt
```
Comment (MadLittleMods) [inline docs/setup/installation.md:157]: This also needs a changelog, we can just create `17846.doc`

```
Update maintainer of Alpine Linux Synapse package.
```

**Comment by jahway603** [comment]
> @jahway603 This is just waiting on the CLA being signed

I've signed the CLA before in this PR https://github.com/element-hq/synapse/pull/17140

File: docs/setup/installation.md (hunk 1 above)
Comment (MadLittleMods) [reply docs/setup/installation.md:157]: @jahway603 Can you add this?

**Comment by MadLittleMods** [comment]
> > @jahway603 This is just waiting on the CLA being signed
> 
> I've signed the CLA before in this PR #17140

hmm, you seem to be right as far as I can tell (same email in the patch for this PR and the one you linked). When I click "recheck" it doesn't seem to do anything.

@devonh (maintainer ping) any idea?

**Comment by devonh** [comment]
> > > @jahway603 This is just waiting on the CLA being signed
> > 
> > 
> > I've signed the CLA before in this PR #17140
> 
> hmm, you seem to be right as far as I can tell (same email in the patch for this PR and the one you linked). When I click "recheck" it doesn't seem to do anything.
> 
> @devonh (maintainer ping) any idea?

It looks like the CLA was updated to v2.0 on October 10, 2024.
@jahway603 you will need to sign the latest version of the CLA.

**Comment by jahway603** [comment]
@devonh how do I do that "safely"? I remember I was able to not dox myself on GitHub last time I signed the CLA in PR #17140.

**Comment by jahway603** [comment]
This PR relates to Issue https://github.com/element-hq/synapse/issues/17977.

**Comment by devonh** [comment]
> @devonh how do I do that "safely"? I remember I was able to not dox myself on GitHub last time I signed the CLA in PR #17140.

Following the link from the CLA Assistant above is "safe". 
The information provided isn't public and is private to Element.

**Review by devonh — APPROVED** [review]

**Comment by devonh** [comment]
Thanks for jumping through all the hoops and getting this changed :)

**Comment by jahway603** [comment]
Fixes Issue https://github.com/element-hq/synapse/issues/17977

**Pull request merged**